metsim/methods/mtclim.py

```diff
@@ -89,72 +88,72 @@ def _calc_tfmax(precip, dtr, sm_dtr):
         warn('missing values with unsmoothed data')
         sm_dtr.fillna(dtr, inplace=True)
 
-    # Calculate annual total precip
-    sum_precip = df['precip'].values.sum()
-    ann_precip = (sum_precip / params['n_days']) * cnst.DAYS_PER_YEAR
-    if ann_precip == 0.0:
-        ann_precip = 1.0
+    # Calculate annual total prec
+    sum_prec = df['prec'].values.sum()
+    ann_prec = (sum_prec / params['n_days']) * cnst.DAYS_PER_YEAR
+    if ann_prec == 0.0:
+        ann_prec = 1.0
```

Comment:
Looks like we can actually get rid of/simplify that section of code entirely.  I'll make an issue about it and address it in a future pull request.  I think it's better to do that way to simplify merging, since @jhamman has done a bunch of cleanup in a separate pull request (where this should belong).